src/rev_ai/models/__init__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""Models"""
+
+from models.job import Job
+from models.job_options import JobSubmitOptions
+from models.error import ApiError, InsufficientBalanceError, InvalidParametersError, InvalidValueError
```

Comment:
if you want to stay in 80 char limits (very doable in python), you can use the tuple with explicit parenthesis pattern (as you use below)